medical-portal/src/UI/src/app/case-details/case-details.component.html

```diff
@@ -97,7 +97,11 @@ <h2>Active Case Details</h2>
         orientation="horizontal"
       >
         <mat-step state="circle">
-          <ng-template matStepLabel>Opened:<br /></ng-template>
+          <ng-template matStepLabel
+            >Opened:<br />{{
```

Comment:
formatting seems weird here, are you putting the `>` on the next line to avoid adding a space before "Opened:"?

Reply:
This is auto formatted using VS code generator